webapp/src/types/gitlab_types.d.ts

```diff
@@ -2,9 +2,9 @@ interface IssueBody {
     title: string;
     description: string;
     project_id?: number;
-    labels?: (string | number)[];
-    assignees?: (string | number)[];
-    milestone?: (string | number);
+    labels?: (string | number | Issue)[];
+    assignees?: (string | number | Issue)[];
+    milestone?: (string | number | Issue);
```

Comment:
Why can all 3 of these be these 3 things? I would think `string` should cover the bases?